configure.ac

```diff
@@ -24,13 +24,32 @@ AC_ARG_ENABLE([unit],
                             [build cmocka unit tests (default is no)])],
             [enable_unit=$enableval],
             [enable_unit=no])
-AS_IF([test "x$enable_unit" != xno],
-      [PKG_CHECK_MODULES([CMOCKA],
-                         [cmocka],
-                         [AC_DEFINE([HAVE_CMOCKA],
-                                    [1])])])
 AM_CONDITIONAL([UNIT], [test "x$enable_unit" != xno])
 
+AC_DEFUN([unit_test_checks],[
+
+    PKG_CHECK_MODULES([CMOCKA],[cmocka],
+        [AC_DEFINE([HAVE_CMOCKA],[1])])
+
+    AC_CHECK_PROG([tpm2_abrmd], [tpm2-abrmd],
+        [yes], [no])
+
+    AC_CHECK_PROG([tpm_server], [tpm_server],
+        [yes], [no])
+
+    AS_IF([test "$tpm2_abrmd" != "yes"],
+           [AC_MSG_ERROR([Required executable tpm2_abrmd not found, try setting PATH])],
+        [])
+
+    AS_IF([test "$tpm_server" != "yes"],
+        [AC_MSG_ERROR([Required executable tpm2_server not found, try setting PATH])],
+        [])
+])
+
+AS_IF([test "$unit" != "no"],
```

Comment:
Should be:
```
AS_IF([test "x$unit" != "xno"],
       [unit_test_checks],
       [])
```

Otherwise, it may be some bug.
I cannot build tpm2-tools on a system without cmocka.

```
checking for CMOCKA... no
configure: error: Package requirements (cmocka) were not met:
No package 'cmocka' found
```

Using `./configure --disable-unit` won't work either.